docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -0,0 +1,96 @@
+.. _2017_cause_models:
+
+============
+Cause Models in GBD
+============
+
+What is a cause?
+----------------
+A cause in the GBD is something that is responsible for death, disease, or disability. 
+Some examples could be infectious diseases like malaria or diarrhea. 
+They could also be chronic diseases like diabetes or cancer. The could be things
+that people are born with like congenital disorders or happen during childbirth like maternal hemorrhage.
+Some causes are only associated with non-fatal outcomes, such as lower back pain.
+Many different things cause health loss and 
+this variation is reflected in the many ways that GBD models them. 
+
+Causes in the GBD are organized in a hierarchy where at every level of that hierarchy
+causes of death or disease are mutually exclusive and collectively exhaustive. 
+
+- All causes
+
+  - Injuries, communicable diseases, non-communicable diseases
+
+    - ... down to the most detailed level
+
+Measures for causes include:  
+Deaths, incidence, prevalence, YLDs, YLLs, and DALYs
+
+And metrics for causes include:  
+Counts, rates, and fractions
+
+Modeling causes of death
+------------------------
+
+.. image:: fatal_flowchart.png
+   :width: 600
+
+**Modeling approach** 
+
+Most causes are estimated using a centrally-maintained Bayesian ensemble modeling tool (CODEm)
+
+- Bayesian: estimates based on probabilities
+- Ensemble: combines many unique submodels into a final model
+
```

Comment:
Paper out today that provides some details on CODEm in a style intended to be understandable:  https://doi.org/10.1093/ije/dyz223